### ISSUE TITLE
Add range slider for letter spacing 

### ISSUE THREAD

**Issue by Abdulmumin1** [Abdulmumin1/Merch-Designer#4]
Add a range slider that increases and reduces letter-spacing of the merch label.

**Comment by kaustav202** [comment]
Let me take this up @Abdulmumin1

**Comment by Abdulmumin1** [comment]
You are always welcome, don't forget the readme instructions to get the projects running.

**Comment by Abdulmumin1** [comment]
@kaustav202 are you still working on this :eyes:!

**Comment by kaustav202** [comment]
yes I am @Abdulmumin1 ..was unwell for last few of days. shall be done by tom.
